### PR TITLE
Adding FreezeTimeMixin and DeclEnum to flask_common

### PR DESCRIPTION
**Pull request by lucasvo**

### CODE REVIEW

File: flask_common/test_helpers.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+from freezegun import freeze_time
+
+class FreezeTimeMixin():
```
Comment (philfreo) [inline flask_common/test_helpers.py]: Docstring + example / description please

File: flask_common/declenum.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# From: http://techspot.zzzeek.org/2011/01/14/the-enum-recipe/
+
+from sqlalchemy.types import SchemaType, TypeDecorator, Enum
+import re
+
+class DeclEnumType(SchemaType, TypeDecorator):
```
Comment (philfreo) [inline flask_common/declenum.py]: Docstring on these classes or at least a higher level description at the top of how to use them in the basic cases

File: flask_common/test_helpers.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    """
+    Helper to use freeze_gun for unit tests.
+    ----
+    Use 1self.freeze(datetime.datetime())` to mock `datetime.datetime.now()` and `self.unfreeze()` to revert the mock.
```
Comment (philfreo) [inline flask_common/test_helpers.py]: typo

Comment (lucasvo) [inline flask_common/test_helpers.py]: fixed.

**Pull request merged**